app/models/video.py

```diff
@@ -0,0 +1,30 @@
+from flask import current_app
+from app import db
+
+def default_avail_inv(context): 
+    return context.get_current_parameters()["total_inventory"]
+
+class Video(db.Model):
+    video_id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
+    release_date = db.Column(db.DateTime, nullable=True)
+    total_inventory = db.Column(db.Integer)
+    available_inventory = db.Column(db.Integer, default=default_avail_inv)
```

Comment:
Like with `Customer` you can compute `available_inventory` with a relationship:

```suggestion
    rentals = db.relationship('Rental', back_populates='video', lazy=True)
```